Remove old migrations

sascha-karnatz opened the pull request:
Remove older migrations that were necessary for an older version of Alchemy. On fresh installation, these migration are causing troubles, because the altered tables are not available anymore for Alchemy >= 7.x.

sascha-karnatz committed:
Remove old migrations

Remove older migrations that were necessary for an older version of Alchemy. On fresh installation, these migration are causing troubles, because the altered tables are not available anymore for Alchemy >= 7.x.

sascha-karnatz committed:
Fix Alchemy version in README

This version of Alchemy PgSearch has a minimum requirement of AlchemyCMS 7.0.

tvdeyen commented:
I think the Dummy app needs an update as well. The old migrations are still included there https://github.com/AlchemyCMS/alchemy-pg_search/tree/main/spec/dummy/db/migrate

sascha-karnatz committed:
Remove nearly all migrations from dummy app

Migrate the dummy fresh with Alchemy 7.0 and remove nearly all migration. Ignore the migrate folder in the gitignore to store only the generated schema.rb. The only exception is the create_user migration that is a custom migration of the dummy app to have the possibility to run user specific feature specs.
